package.json

```diff
@@ -46,6 +46,7 @@
     "bin/**/*",
     "!src/**/*",
     "!**/test",
+    "!**/benchmarks",
```

Comment:
aren't these already ignored by `"!**/test"` above? benchmarks should only live inside tests/

Reply:
I run `npm pack` and get this without those changes: 🤔 
<img width="324" alt="Screenshot 2025-01-23 at 13 35 50" src="https://github.com/user-attachments/assets/fec8789d-6b02-4ef3-889a-a6b6198fc67c" />

Reply:
you should remove `./src/benchmarks` locally, it's some leftovers most likely

Reply:
I thought so too, but I checked without `src/benchmarks` and got the same result

Reply:
Oops, I need to delete the `dist/` folder